open_xdmod/modules/xdmod/integration_tests/email-subject-test.sh

```diff
@@ -5,6 +5,16 @@ curl "${host}controllers/mailer.php" --compressed -H 'X-Requested-With: XMLHttpR
 
 sed -i -- 's/subject_prefix = "SEND BEN MONEY"/subject_prefix = ""/' /etc/xdmod/portal_settings.ini
 
+# sometimes it takes a second to send the emails
+queueLength=`postqueue -p | tail -n1 | awk '{print $5}'`
+count=0
+while [[ "$queueLength" != ""  && $count < 5 ]]; do
```

Comment:
If you're going to add a count then you should also exit with failure if the max count is reached and output a message explaining that the test failed due to the count (i.e. that the queue never emptied) and not because the email subject wasn't found (because the email was never received).

Also, `-lt` should be used in place of `<` for integer comparisons.  (It doesn't cause a problem when you're using `5`, but if you changed that to `10` it wouldn't work as expected.

```suggestion
while [[ "$queueLength" != ""  && $count -lt 5 ]]; do
```